### PR TITLE
more refactoring

### PR DESCRIPTION
**Pull request by biomunky**

### CODE REVIEW

**Comment by rsyi** [comment]
Hey @biomunky ! This also looks good to me :) Let me know when it's good to go!

**Comment by biomunky** [comment]
👍 there's a load of stuff i need to do but currently got some crap going on here so will be in and out infrequently. I've also DM-ed you on Slack about the refactoring from last time.

**Comment by biomunky** [comment]
I tested this against a local postgres docker instance and can see tables when I run whale - please double check. I will probably start adding tests and stop with the yak shaving.

File: cli/src/skimmer.rs [hunk 1]
```diff
@@ -7,79 +7,85 @@ use std::io::Cursor;
 use std::path::Path;
 use std::process::Command;
 
-pub fn table_skim() {
-    let manifest_file_path = filesystem::get_manifest_filename();
-    let tmp_manifest_file_path = filesystem::get_tmp_manifest_filename();
+const ASCII_ART: &str = "<< EOF
+___=____
```
Comment (rsyi) [inline cli/src/skimmer.rs:11]: This is really, really a nit @biomunky but the top of the whale is now 4 spaces too far to the left. Can you move it back? lol

**Review by rsyi — APPROVED** [review]
LGTM and runs smoothly. added one nit, but if you don't get to it, I'll just fix and merge :P

**Comment by rsyi** [comment]
NIIIIIIIICCEEEEEEEEEE

**Comment by rsyi** [comment]
huh. clippy failed on this run. weird. mind checking that too @biomunky?

**Comment by biomunky** [comment]
> huh. clippy failed on this run. weird. mind checking that too @biomunky?

clippy has been updated and has some good suggestions, I need to update my local setup.

**Comment by rsyi** [comment]
good callout @biomunky. thanks for making the changes anyway -- in the future i'll do a more careful check before requesting edits.

**Pull request merged**